analytics/ga.go

```diff
@@ -35,6 +35,7 @@ type UserSession struct {
 	TrackingID         string       `url:"tid"`
 	ApplicationName    string       `url:"an"`
 	ApplicationVersion string       `url:"av"`
+	DockerAPIVersion   string       `url:"cd1"`
```

Comment:
Does `cdl` have a specific meaning? Rest of the keys map to the variable name pretty obviously?